tests/Spout/Writer/XLSX/SheetTest.php

```diff
@@ -131,4 +163,19 @@ private function assertSheetNameEquals($expectedName, $fileName, $message = '')
 
         $this->assertContains("<sheet name=\"$expectedName\"", $xmlContents, $message);
     }
+
+    /**
+     * @param string $expectedVisibility
+     * @param string $fileName
+     * @param string $message
+     * @return void
+     */
+    private function assertSheetVisibilityEquals($expectedVisibility, $fileName, $message = '')
```

Comment:
This function is confusing... First it does not use `$expectedVisibility`. Also, it does a global search across the workbook.xml file. This means that if a file has 2 sheets, one hidden, one visible, this function will always return `true`.